header.php

```diff
@@ -80,71 +88,47 @@
   <!-- start content -->
   <header id="header" class="section">
     <div class="container">
-      <div class="row">
-        <div class="col col-l col-l-2 flex-col justify-center align-center">
+      <div class="row font-uppercase">
+        <div class="col col-l col-l-5 flex-col justify-center align-center">
         <?php if (!empty($logo_id)) { ?>
           <a href="<?php echo home_url(); ?>"><?php echo wp_get_attachment_image($logo_id, 'thumbnail'); ?></a>
```

Comment:
should that not be in skin front page tho? if not then imo skin header should have come first separate and same for skin footer, then skin front page